### ISSUE TITLE
Semantic conventions for hardware metrics appears blank

### ISSUE THREAD

**Issue by bertysentry** [open-telemetry/opentelemetry-specification#3013]
Page https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/hardware-metrics/ is rendered as an empty page:
![image](https://user-images.githubusercontent.com/32521698/205900485-b71bc282-48a6-4033-9142-6c78c54d8e70.png)

Content that is expected: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/hardware-metrics.md

**Comment by arminru** [comment]
@open-telemetry/docs-maintainers could you take a look please?
We can also transfer this to the website repo, not sure where exactly this needs to be fixed.

**Comment by svrnm** [comment]
Yes, please move this issue over to opentelemetry.io repo, we will look into that.
